crates/usvg/src/tree/text.rs

```diff
@@ -565,7 +565,20 @@ impl Text {
         self.abs_stroke_bounding_box
     }
 
-    /// Text converted into paths, ready to render.
+    /// Text converted into paths, ready to render. Note that this is only a
```

Comment:
Can you format it a bit better? Check out `cargo docs` output. It would be formatted as a one long line.

Reply:
Not sure I follow, this is how it looks for me, what exactly is wrong with that? Should I add a new line after the first sentence?

<img width="1029" alt="image" src="https://github.com/RazrFalcon/resvg/assets/47084093/dd772b01-0533-4175-8499-eee00b4be71f">

Reply:
>Should I add a new line after the first sentence?

Yes.

Reply:
Done.